docs/lib/restapi/fragments/android/fetch.md

```diff
@@ -0,0 +1,62 @@
+## GET requests
+
+```java
+void getTodo() {
+    RestOptions options = new RestOptions("/todo");
+    Amplify.API.get(options,
+            restResponse -> Log.i("ApiGettingStarted", restResponse.toString()),
+            apiFailure -> Log.e("ApiGettingStarted", apiFailure.getMessage(), apiFailure)
+    );
+}
+```
+
+## Accessing query parameters & body in Lambda proxy function
+
+> To learn more about Lambda Proxy Integration, please visit [Amazon API Gateway Developer Guide](https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-create-api-as-simple-proxy-for-lambda.html).
+
+If you are using a REST API which is generated with Amplify CLI, your backend is created with Lambda Proxy Integration, and you can access your query parameters & body within your Lambda function via the *event* object:
+
+```javascript
+exports.handler = function(event, context, callback) {
+    console.log (event.queryStringParameters);
+    console.log('body: ', event.body);
+}
+```
+
+Alternatively, you can update your backend file which is located at `amplify/backend/function/[your-lambda-function]/app.js` with the middleware:
+
+```javascript
+const awsServerlessExpressMiddleware = require('aws-serverless-express/middleware');
+app.use(awsServerlessExpressMiddleware.eventContext());
+```
+
+Accessing Query Parameters with Serverless Express
+
+In your request handler use `req.apiGateway.event` or `req.query`:
+
+```javascript
+app.get('/todo', function(req, res) {
+  const query = req.query;
+  // or
+  // const query = req.apiGateway.event.queryStringParameters
+  res.json({
+    event: req.apiGateway.event, // to view all event data
+    query: query
+  });
+});
+```
+
+Then you can use query parameters in your path as follows:
+
```

Comment:
I feel like there is an ordering problem here...  Probably doens't need to be before GA, but this section seems like it should be ordered in the following manner:'
* Start with in with iOS/Android example of how to pass in query parameters
* Where to update the lambda ( the 2nd example where we reference awsServerlessExpressMiddleware, it is obvious, but unclear where to update the 1st and 3rd code samples)